client/src/components/imagine/TicTacToe/TicTacToeBoard.jsx

```diff
@@ -29,59 +29,62 @@ const TicTacToeBoard = (props)=> {
         setGameState({...Model.checkWinner(gameInfo)});
         // swaps players
         setActivePlayer(Model.Players[0]);
+
         };
     
 
     return(
-        <div>
-            <div className="tw-container tw-content-center tw-pb-5">
-                {
-                    gameState.isGameOver === Model.GameState.isGameOver ?// <h2> It's the {activePlayer.name}'s turn </h2>:
-                    <></> : (gameState.isGameOver ? <h2>The {gameState.winner} is the winner</h2>:
-                    <h2> The Game was a Tie</h2>)
-                }
-            </div>
-            <div className="tw-container tw-bg-dark tw-mx-auto tw-space-y-2 md:tw-space-y-0 md:tw-gap-2 md:tw-grid md:tw-grid-cols-3 lg:tw-grid-cols-3 tw-p-3">
-                <div>
+        <>
+        <div className="moduleContainer">
```

Comment:
I like where you went with this more than what I had implemented for functionality in the first place. it makes more sense and gives the user a better prespective.